both/methods.js

```diff
@@ -228,11 +228,11 @@ Meteor.methods({
         registrationId: 1,
       },
     }).fetch();
-    if(competitorCount < 0) {
+    if(participantCount < 0) {
```

Comment:
This comment might not be in the right place, but I wanted point it out since I saw it: Shouldn't this actually be < 1, as you don't want to advance 0, or even a partial participant to the next round?